src/ui/components/Header/UserOptions.js

```diff
@@ -14,7 +14,7 @@ function UserOptions({ recordingId, setModal }) {
   const [expanded, setExpanded] = useState(false);
   const { isAuthenticated } = useAuth0();
 
-  const showShare = hooks.useIsOwner(recordingId);
+  const showShare = recordingId ? hooks.useIsOwner(recordingId) : false;
```

Comment:
yikes... i dont think we should put a condition around a hook... is there another way of doing this like?

```js
const showShare = hooks.useIsOwner(recordingId ? recordingId : false);
```

Reply:
Whoops, of course we shouldn't. It feels wrong to send a dummy query to Hasura because of this, but I can't think of a way to avoid this...